Enable pseudolocalization from pseudolocale folder (#836)
lingui/js-lingui: docs/tutorials/cli.rst

:conf:`pseudoLocale` option can be any string 
examples: :conf:`en-PL`, :conf:`pseudo-LOCALE`, :conf:`pseudolocalization` or :conf:`en-UK`

PseudoLocale string have to be in `locale` config as well. Otherwise no folder is going to be created.
Pseudolocalized text is created on  ``yarn compile`` command.

> The pseudolocalization is automatically created from TODOTODOTODO.

Author: Do we have already in documentation what is the prioritisation of compiling messages?

Reviewer: I don't think so. It's:

1. translation
2. fallback (using the new `fallbackLocales`)
3. message key

Author: Wouldn't it be worth adding it to documentation? I just can't figure out where it could fit

Reviewer: Yeah, absolutely. Somewhere inside `compile` docs? Because it ultimately happens at compile step.

:conf:`pseudoLocale` option can be any string 
examples: :conf:`en-PL`, :conf:`pseudo-LOCALE`, :conf:`pseudolocalization` or :conf:`en-UK`

> PseudoLocale string have to be in `locale` config as well. Otherwise no folder is going to be created.

Reviewer: ```suggestion
PseudoLocale string have to be in :conf:`locales` config as well. Otherwise no folder is going to be created.
```

Pseudolocalized text is created on  ``yarn compile`` command.

> The pseudolocalization is automatically created from messages in order specified in "LINK TO Preparing catalogs for production".

Author: How can I link to added compile section above?

Reviewer 2: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Reviewer: Every heading is an anchor that you can reference with `:ref:`:

```suggestion
The pseudolocalization is automatically created from messages in order specified in :ref:`Preparing catalogs for production`.
```

You could also customize the link if you want to spell it differently:

```
The pseudolocalization is automatically created from messages in order specified in :ref:`section above<Preparing catalogs for production>`.
```

> It can also be changed by setting translation in :conf:`message.json` into non-pseudolocalized text.

Reviewer: Not sure what link was supposed to be here but I guess it should be sth like this:

```suggestion
It can also be changed by setting translation in catalog into non-pseudolocalized text.
```

Author: I think we should remove this sentence. It was supposed to show that users are able to edit message.json file manually. Considering that it would get removed after running compile then I do not find it useful.

Author: I have removed the sentence
